lib/SparkPost/SparkPost.php

```diff
@@ -26,7 +26,8 @@ class SparkPost {
       'port'=>443,
       'strictSSL'=>true,
       'key'=>'',
-      'version'=>'v1'
+      'version'=>'v1',
+      'subaccount_id' => 0
```

Comment:
Thanks for checking this for me. So the 0 is just getting falsy checked. But I wanted to make sure the key was there otherwise key_exists in setConfig won't set it at all. 

Would null be better?